runtime/Makefile.help

```diff
@@ -138,7 +140,7 @@ $(RUNTIME_LIB): $(RUNTIME_OBJS) $(RUNTIME_DIR_TIMESTAMP) $(RUNTIME_LIST_INCLUDES
 	$(AR) -rc $@ $(RUNTIME_OBJS)
 	$(RANLIB) $@
 	$(TAGS_COMMAND)
-	../util/printchplenv --simple > $(RUNTIME_CHPLCONFIG)
+	../util/printchplenv --all --no-tidy --anonymize --simple > $(RUNTIME_CHPLCONFIG)
```

Comment:
Like the previous runtime one, since this is generating a chplconfig stored in the *runtime* directory, it shouldn't include the launcher or compiler settings. So I think this should use --runtime instead of --all.